cmd/fleet/main.go

```diff
@@ -68,7 +68,26 @@ func makeCache(cfg *config.Config) (cache.Cache, error) {
 	return cache.New(cacheCfg)
 }
 
-func getRunCommand(version string) func(cmd *cobra.Command, args []string) error {
+func initLogger(cfg *config.Config, version, commit string) (*logger.Logger, error) {
+	l, err := logger.Init(cfg)
+	if err != nil {
+		return nil, err
+	}
+
+	log.Info().
+		Str("version", version).
+		Str("commit", commit).
+		Int("pid", os.Getpid()).
+		Int("ppid", os.Getppid()).
+		Str("exe", os.Args[0]).
+		Strs("args", os.Args[1:]).
```

Comment:
Could these args contain confidential info?

Reply:
Question for @blakerouse .  The args are currently being logged with or without this change. We can remove them, but I am hesitant as they are very useful for debugging.

Reply:
The args do not container any credentials in them. All credentials are passed over GRPC and the certificates for the mTLS for local GRPC is done through Stdin. I don't see any danger keeping the arguments, and they are not to have for debugging.